app/src/main/java/com/example/househomey/HomeFragment.java

```diff
@@ -44,10 +44,8 @@ public class HomeFragment extends Fragment implements FilterCallback {
 
     /**
      * This constructs a new HomeFragment with the appropriate list of items
-     * @param itemRef A reference to the firestore collection containing the items to display
      */
-    public HomeFragment(CollectionReference itemRef) {
-        this.itemRef = itemRef;
+    public HomeFragment() {
```

Comment:
If the Constructors for fragments are bad, let's just move this line to the beginning of onCreateView and get rid of the constructor.